b9lab-content/5-my-own-chain/03-starport.md

```diff
@@ -1 +1,198 @@
-# Starport
\ No newline at end of file
+# Starport
+
+## Install
+
+One very easy way to create Blockchain applications with the Cosmos SDK is to use Starport. You can install it via:
+
+```bash
+$ curl https://get.starport.network/starport! | bash
+```
+
+In case you face permission issues for writing into `usr/local/bin`, you can move the starport executable manually with:
+
+```bash
+$ sudo mv starport /usr/local/bin/
+```
+
+And verify your installation with a version request:
+
+```bash
+$ starport version
+```
+
+You can also just type `starport` to see the commands it offers:
+
+```bash
+Starport is a tool for creating sovereign blockchains built with Cosmos SDK, the world’s
+most popular modular blockchain framework. Starport offers everything you need to scaffold,
+test, build, and launch your blockchain.
+
+To get started, create a blockchain:
+
+starport scaffold chain github.com/cosmonaut/mars
+
+Usage:
+  starport [command]
+
+Available Commands:
+  scaffold    Scaffold a new blockchain, module, message, query, and more
+  chain       Build, initialize and start a blockchain node or perform other actions on the blockchain
+  generate    Generate clients, API docs from source code
+  network     Launch a blockchain network in production
+  relayer     Connects blockchains via IBC protocol
+  tools       Tools for advanced users
+  docs        Show Starport docs
+  version     Print the current build information
+  help        Help about any command
+
+Flags:
+  -h, --help   help for starport
+
+Use "starport [command] --help" for more information about a command.
+```
+
+There is a very good [introduction tutorial](https://docs.starport.network/guide/hello.html) provided by the [Starport documentation](https://docs.starport.network/) which you should first check and try to see the Hello-World response at [http://localhost:1317/cosmonaut/hello/hello/posts](http://localhost:1317/cosmonaut/hello/hello/posts). 
+
+## Your chain
+
+You can create a chain for Alice with:
+
+```bash
+$ starport scaffold chain github.com/alice/chain
+```
+
+The scaffolding will take some time. After the scaffolding you will have a folder `chain`. 
+In it you will see that we have 
+  - a folder for the application called `app`, 
+  - a folder for the commands called `cmd`, 
+  - a folder for the protobuff definitions called `proto`,
+  - a folder for the UI called `vue`, see [Vue.js](https://vuejs.org/).
```

Comment:
```suggestion
The scaffolding takes some time as it generates the source code for a fully functional ready-to-use blockchain. After the chain is scaffolded, you have a folder `chain`. 

The `chain` folder contains a number of generated files and directories that make up the structure of a Cosmos SDK blockchain:

  - A folder for the application called `app`
  - A folder for the commands called `cmd`
  - A folder for the protobuff definitions called `proto`
  - A folder for the UI called `vue`, see [Vue.js](https://vuejs.org/)
```

Reply:
hi B9lab friends! A few points here as we develop this awesome content:

- Use present tense https://developers.google.com/style/tense 
- Use "you" audience consistently ("we" doesn't have a place here, we are empowering our users) https://developers.google.com/style/person 
- Capitalize the first letter in list items
- Do not include punctuation at the end of list items that are sentence fragments

keep up the good work with the technical bits, let's strive to build trust with consistent high quality grammar too. Questions? please ask